packages/ui/src/components/va-config/hooks/useLocalConfig.ts

```diff
@@ -0,0 +1,17 @@
+import { inject, provide, computed, ComputedRef } from 'vue'
+import { ComponentConfig } from '../../../services/component-config'
+
+/**
+ * We need another key to provide injected value.
+ */
+export const LocalConfigKey = 'VaLocalConfig'
```

Comment:
Why not `Symbol`?

Reply:
Web compoent's provide doesn't support Symbol.